### PR TITLE
Sidecars

### PR DESCRIPTION
**Pull request by Olivier-St-Pierre**

### CODE REVIEW

File: charts/redis/values.yaml [hunk 1]
```diff
@@ -54,6 +54,8 @@ redisExporter:
   #       name: mysecret
   #       key: username
 
+sidecars: []
```
Comment (devodev) [inline charts/redis/values.yaml:57]: We should document the possible fields in a comment, similar to above.

File: charts/redis-cluster/values.yaml [hunk 2]
```diff
@@ -63,6 +63,8 @@ redisExporter:
     #   cpu: 100m
     #   memory: 128Mi
 
+sidecars: []
```
Comment (devodev) [inline charts/redis-cluster/values.yaml:66]: same as my other comment about documentation.

**Review by devodev — CHANGES_REQUESTED** [review]
Please document possible values in values files.

**Review by devodev — APPROVED** [review]
LGTM :)

**Comment by Olivier-St-Pierre** [comment]
Do I wait for a 2nd review or I merge?

**Pull request merged**